dotnet-desktop-guide/net/wpf/documents/snippets/printing-overview/vb/MainWindow.xaml.vb

```diff
@@ -0,0 +1,170 @@
+﻿Imports System.IO
+Imports System.Printing
+Imports System.Windows.Xps
+Imports System.Windows.Xps.Packaging
+Imports Microsoft.Win32
+
+Namespace CodeSampleVb
+    Partial Public Class MainWindow
+        Inherits Window
+
+        ''' <summary>
+        ''' Interaction logic for MainWindow.xaml
+        ''' </summary>
+        Public Sub New()
+
+            InitializeComponent()
+
+        End Sub
+
+        Private Sub Grid_Loaded(sender As Object, e As RoutedEventArgs)
+
+            For Each printQueue As PrintQueue In GetPrintQueues()
+                '' Add printers that have at least one relevant print options.
```

Comment:
VB code comments are a single ' not a double '' . I would change these to make it more VB-like

Reply:
Done.